infra/nightly-resources/chart.js

```diff
@@ -16,7 +16,7 @@ const BASELINE_MODE = "llvm-O0";
 
 // Given a list of integers, compute the mean
 // number of cycles
-function mean_cycles(cycles) {
+function mean(cycles) {
```

Comment:
nit: since this fn isn't actually specific to cycles, can the param/comment etc just talk about mean of a list of ints generally?